crates/sargon/src/types/samples/account_address_samples.rs

```diff
@@ -0,0 +1,23 @@
+use crate::prelude::*;
+
+pub(crate) static ALL_ACCOUNT_ADDRESSES_SAMPLES: Lazy<[AccountAddress; 10]> =
+    Lazy::new(|| {
+        [
+            AccountAddress::random(NetworkID::Mainnet),
+            AccountAddress::random(NetworkID::Mainnet),
+            AccountAddress::random(NetworkID::Mainnet),
+            AccountAddress::random(NetworkID::Mainnet),
+            AccountAddress::random(NetworkID::Mainnet),
+            AccountAddress::random(NetworkID::Mainnet),
+            AccountAddress::random(NetworkID::Mainnet),
+            AccountAddress::random(NetworkID::Mainnet),
+            AccountAddress::random(NetworkID::Mainnet),
+            AccountAddress::random(NetworkID::Mainnet),
+        ]
+    });
+
+impl AccountAddress {
+    pub fn sample_at(index: usize) -> Self {
+        ALL_ACCOUNT_ADDRESSES_SAMPLES[index].clone()
```

Comment:
hmm is `0` and `1` index correct for:

```rust
       // Alice | 0 | Unsecurified { Device }
        Account::sample_unsecurified_mainnet(
            "Alice",
            HierarchicalDeterministicFactorInstance::sample_fia0(),
        ),
        // Bob | 1 | Unsecurified { Ledger }
        Account::sample_unsecurified_mainnet(
            "Bob",
            HierarchicalDeterministicFactorInstance::sample_fia1(),
        ),
```

ie, if you do `assert_eq!(Account:: sample_at(0).address(), AccountAddress::sample_at(0));` does that unit test pass?